api/godog/requests.go

```diff
@@ -108,6 +109,10 @@ func (api *BaseFeature) SetRequestBodyStringListParameterTo(key, valuesstr strin
 }
 
 func (api *BaseFeature) ExecuteTheRequest() (err error) {
+	defer func() {
+		log.Debugf("Response: %s", api.Response.Body)
+	}()
+
```

Comment:
Maybe move this to ExecuteTheRequestWithPayload, so we get the debug logging from direct calls to that aswell.